File: LeetCode_Marathons/LCM1/Easy/Excel_Sheet_Column_Title/a.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder ans = new StringBuilder();

        while (columnNumber != 0) {
            columnNumber--;
            ans.insert(0, (char) ('A' + columnNumber % 26));
            columnNumber /= 26;
        }

        return ans.toString();
    }
}

File: LeetCode_Marathons/LCM1/Easy/Excel_Sheet_Column_Title/a.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string ans;

        while (columnNumber != 0) {
            columnNumber--;
            ans = char( int('A') + columnNumber % 26 ) + ans;
            columnNumber = columnNumber / 26;
        }

        return ans;
    }
};